website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,44 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroying infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Deleting a workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all of a workspace's managed infrastructure.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. We recommend destroying the infrastructure managed by a workspace _before_ deleting the workspace itself. Otherwise, the unmanaged infrastructure resources will continue to exist but will become unmanaged, and you must go into your infrastructure providers to delete the resources manually.
+
+Before queuing a destroy plan, you must enable the **Allow destroy plans** toggle setting on this page.
+
+### Automatically Destroy
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/ephemeral-workspaces.mdx'
+<!-- END: TFC:only name:pnp-callout -->
+
+Automated destroy runs are available as part of the ephemeral workspaces feature.
+
+Some use cases may benefit from allowing Terraform Cloud to automatically schedule infrastructure deletion at some point in the future. For example, you may want to automatically destroy a workspace's resources used solely in a development environment after a certain period of time for cost savings purposes.
+
+You can automatically schedule Terraform Cloud to run a destroy at a specific future date and time.
+
+When Terraform Cloud performs the destroy run this setting will be automatically unset. You can re-enable it to schedule another future destroy run.
```

Comment:
I like your original phrasing, but I think we can make it even more clear with something like:
```suggestion
This setting is automatically unset whenever Terraform Cloud performs a destroy run. You can re-enable it to schedule another future destroy run.
```